netZooPy/panda/calculations_gpu.py

```diff
@@ -79,7 +91,7 @@ def compute_panda_gpu(
     ppi_matrix = cp.array(ppi_matrix.copy())
     motif_matrix = cp.array(motif_matrix.copy())
     correlation_matrix = cp.array(correlation_matrix.copy())
-
+    
     while hamming > threshold:
 
         W = 0.5 * (
```

Comment:
so here for example it should be float32(0.5), that's what I meant last time that each float has to be single because the product of single and double is double. You can leave integers as they are

Reply:
I checked and W is float32 in this case (when precision is single)

Reply:
this is interesting because
>>> 0.5*np.float32(0.54387218494723)
0.2719360888004303
>>> np.float32(0.5)*np.float32(0.54387218494723)
0.2719361

Reply:
In the case you tested you are right, but it looks like for arrays the behaviour is different: ``